server/node/src/server.ts

```diff
@@ -74,6 +78,35 @@ app.post(
   },
 );
 
+app.post(
+  "/paypal-api/vault/payment-token/create",
+  async (req: Request, res: Response) => {
+    try {
+      const { jsonResponse, httpStatusCode } = await createPaymentToken(
+        req.body.vaultSetupToken as string,
+      );
+
+      const paymentTokenResponse = jsonResponse as PaymentTokenResponse;
+
+      if (paymentTokenResponse.id) {
+        await savePaymentTokenToDatabase(paymentTokenResponse);
```

Comment:
I think its important that we teach developers NOT to return this paymentToken value to the front-end. This is a long-lived value that can be used for making payments where the buyer is not present or doing a one-shot transaction.

We want to teach developers to save this payment token in their database instead.